weblate/accounts/templatetags/authnames.py

```diff
@@ -48,6 +49,15 @@
     'facebook': {'name': 'Facebook', 'fa_icon': 'facebook'},
     'twitter': {'name': 'Twitter', 'fa_icon': 'twitter'},
     'stackoverflow': {'name': 'Stack Overflow', 'fa_icon': 'stackoverflow'},
+    'auth0': {
+      'name': getattr(settings, 'SOCIAL_AUTH_AUTH0_TITLE', 'Auth0'),
+      'image': getattr(
+          settings,
+          'SOCIAL_AUTH_AUTH0_IMAGE',
+          'btn_auth0_badge.png'
+          ),
```

Comment:
Please set default values using appconf and you can then use just `settings.SOCIAL_AUTH_AUTH0_IMAGE` here.

https://github.com/WeblateOrg/weblate/blob/54fa2346bb41cffdbc5173439acba93fd15b0c70/weblate/accounts/models.py#L566-L588

Reply:
OK, looks much nicer, thanks for this hint. 

Fixed.